docs/user-guide/references.rst

```diff
@@ -96,6 +146,11 @@ Glossary
       by contrast with the energy reconstructed by the instrument. Instrument response
       functions are represented along a true energy axis.
 
+    TS
+      Short for "test statistics": TS is equal to 2*log(L) with L the likelihood value of
```

Comment:
Note that we rather use delta TS = -2 log L, and I must say that this is not very clear from the fit-statistic page.
Maybe it is better not to include that in the glossary but rather refer to fit-statistics simply to avoid multiple different definitions

Reply:
So, no text only the reference to the (unclear) page? Just tell me
I have added the minus sign, at least...

Reply:
Delta TS is not rather max L(H1)/ max L(H0)?

Reply:
+1 To describe it more detailed on the stats or estimator page and just put references elsewhere...